webapp/src/index.css

```diff
@@ -1,13 +1,118 @@
+/* bring in normalize.css styles */
+@import-normalize;
+
+:root {
+  --color-warning: #ff3a6a;
+  --color-submit: #4b52ee;
+  --color-online: #3fc6a2;
+  --color-offline: #dfc04d;
+  --color-light: #ffffff;
+  --color-background: #191b1f;
+  --color-dark: #262934;
+  --color-game: linear-gradient(
+    to bottom,
+    #4b52ee 0%,
+    #fb4e78 64.06%,
+    #3fc6a2 100%
+  );
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html,
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+}
+
+/* dm-sans-regular - latin */
+@font-face {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: url('./assets/fonts/dm-sans-v11-latin-regular.woff2') format('woff2'),
+    url('./assets/fonts/dm-sans-v11-latin-regular.woff') format('woff');
+}
+
+/* dm-sans-500 - latin */
+@font-face {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url('./assets/fonts/dm-sans-v11-latin-500.woff2') format('woff2'),
```

Comment:
What do you think about, instead of us storing the fonts, using Google Fonts :yum:  -> https://fonts.google.com/specimen/DM+Sans?query=dm+sans

Reply:
We can import this in `<head> -> <link>` https://github.com/42AGV/ft_transcendence/blob/c9e78797fd34c6d6fb162f590c6458b662cc4de4/webapp/public/index.html